.gitignore

```diff
@@ -14,6 +14,7 @@
 /coverage
 /spec/reports
 spec/examples.txt
+.byebug_history
```

Comment:
Should probably just be it's own commit

Reply:
yeah, I've removed this change & rebased.